### PR TITLE
DCA-314: Evaluate and cleanup results dir files

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-413?atlLinkOrigin=c2xhY2staW50ZWdyYXRpb258aXNzdWU%3D

### CODE REVIEW

File: app/util/cleanup_result_dir.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
+files_to_remove = ['jmeter.out',
+                   'jmeter-bzt.properties',
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+
+for _ in files_to_remove:
```
Comment (astashys) [inline app/util/cleanup_result_dir.py]: please name this var for example `file`

Comment (mmizin) [reply app/util/cleanup_result_dir.py]: Done

File: app/util/cleanup_result_dir.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+files_to_remove = ['jmeter.out',
+                   'jmeter-bzt.properties',
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+
+for file in files_to_remove:
+    file_path = Path(artifacts_dir + f'/{file}')
```
Comment (ometelytsia) [inline app/util/cleanup_result_dir.py]: why not to use `Path(f"{artifacts_dir}/{file}")`

File: app/util/cleanup_result_dir.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+import os
+
+artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
+files_to_remove = ['jmeter.out',
```
Comment (ometelytsia) [inline app/util/cleanup_result_dir.py]: Please capitalize constant https://www.python.org/dev/peps/pep-0008/#constants

File: app/util/cleanup_result_dir.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+import os
+
+artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
```
Comment (ometelytsia) [inline app/util/cleanup_result_dir.py]: Pls add here check if variable set to simplify further debugging:
```
ENV_TAURUS_ARTIFACT_DIR = 'TAURUS_ARTIFACTS_DIR'

artifacts_dir = os.getenv(ENV_TAURUS_ARTIFACT_DIR)
if artifacts_dir is None:
    raise SystemExit(f'Error: env variable {ENV_TAURUS_ARTIFACT_DIR} is not set')
```

File: app/bitbucket.yml [hunk 5]
```diff
@@ -26,6 +26,7 @@ services:
       - python util/jtl_convertor/jtls-to-csv.py kpi.jtl selenium.jtl
     post-process:
       - python util/analytics.py bitbucket
+      - python util/cleanup_result_dir.py
```
Comment (ometelytsia) [inline app/bitbucket.yml]: Let's rename the script to `cleanup_results_dir.py`

File: app/util/cleanup_result_dir.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+
+for file in files_to_remove:
+    file_path = Path(artifacts_dir + f'/{file}')
+    try:
+        os.remove(file_path)
+    except OSError as e:
+        print(f'Deleting of the {file} failed!'
```
[the remark this answers is not in the document]
Comment (mmizin) [reply app/util/cleanup_result_dir.py]: Thx, already add

File: app/util/cleanup_result_dir.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+
+for file in files_to_remove:
+    file_path = Path(artifacts_dir + f'/{file}')
+    try:
+        os.remove(file_path)
```
[the remark this answers is not in the document]
Comment (mmizin) [reply app/util/cleanup_result_dir.py]: Do you mean 'logger.debug' in except or something else?

[the remark this answers is not in the document]
Comment (ometelytsia) [reply app/util/cleanup_result_dir.py]: `print()` would be enough. This way bzt.log will have a message.

[the remark this answers is not in the document]
Comment (mmizin) [reply app/util/cleanup_result_dir.py]: Oh, I understood. Thx for explanation

File: app/util/cleanup_result_dir.py (hunk 2 above)
Comment (mmizin) [reply app/util/cleanup_result_dir.py]: Yes your variant looks prettier nice

File: app/bitbucket.yml (hunk 5 above)
Comment (mmizin) [reply app/bitbucket.yml]: Yes, you are right

File: app/util/cleanup_results_dir.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+ENV_TAURUS_ARTIFACT_DIR = 'TAURUS_ARTIFACTS_DIR'
+FILES_TO_REMOVE = ['jmeter.out',
+                   'jmeter-bzt.properties',
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+if ENV_TAURUS_ARTIFACT_DIR in os.environ:
+    artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
```
Comment (ometelytsia) [inline app/util/cleanup_results_dir.py:13]: Pls use here variable ENV_TAURUS_ARTIFACT_DIR

Comment (mmizin) [reply app/util/cleanup_results_dir.py:13]: It is ridiculous, what is wrong with me? ))) I need to be more collected

**Pull request merged**